charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -5,6 +5,12 @@ groups:
   - record: seed:container_cpu_usage_seconds_total:sum_by_container
     expr: sum(rate(container_cpu_usage_seconds_total[5m])) by (container)
 
+  - record: seed:container_cpu_cfs_throttled_seconds_total:sum_by_container
+    expr: sum(rate(container_cpu_cfs_throttled_seconds_total[5m])) by (container)
```

Comment:
Does is make sense to aggregate the rate? `container_cpu_cfs_throttled_seconds_total` is not about how much CPU cores are throttled, right?